cmake/toolchain/cross-compile/Kconfig

```diff
@@ -3,10 +3,4 @@
 
 config TOOLCHAIN_CROSS_COMPILE
 	def_bool y
-
-config TOOLCHAIN_CROSS_COMPILE_SUPPORTS_THREAD_LOCAL_STORAGE
```

Comment:
Why are you removing this ?

This is for the cross-compile toolchain, in which case you don't know if a particular toolchain has been built with / without TLS support.

Therefore this setting in this particular case is having a prompt so that the user of this, potentially self-compiled, toolchain has the ability to specify whether or not the toolchain was built with TLS support.

Reply:
oops, that was an oversight.